src/main/kotlin/no/nav/familie/ef/søknad/mapper/FeltMapperUtil.kt

```diff
@@ -28,3 +29,12 @@ fun dokumentfelt(dokumentnavn: String, vedleggMap: Map<String, DokumentasjonWrap
 data class DokumentasjonWrapper(val label: String, val harSendtInnTidligere: Søknadsfelt<Boolean>, val vedlegg: List<Vedlegg>)
 
 fun String.tilHeltall(): Int = this.toDouble().toInt()
+
+fun TekstFelt.tilSøknadsDatoFeltEllerNull(): Søknadsfelt<LocalDate>? =
+        if (this.verdi.isNotBlank()) {
+            Søknadsfelt(this.label, LocalDate.parse(this.verdi, DateTimeFormatter.ISO_DATE))
```

Comment:
LocalDate.parse??? Andre forslag?

Reply:
`LocalDate.parse` er nok riktig, men har vi tester som sjekker forskjellige varianter av input?
Eks:
2020-06-21
2020-06-21T19:25:21.752Z

Begge disse ser ut som de kommer inn fra tid til annen. 
Ser at denne brukes i Jackson sine LocalDateSerializer 

```
if (format == DEFAULT_FORMATTER && string.length() > 10 && string.charAt(10) == 'T') {
                        return string.endsWith("Z") ? LocalDateTime.ofInstant(Instant.parse(string), ZoneOffset.UTC).toLocalDate() : LocalDate.parse(string, DateTimeFormatter.ISO_LOCAL_DATE_TIME);
                    } else {
                        return LocalDate.parse(string, format);
                    }
```

Reply:
Synes det var veldig omfattende tester for å teste denne, kan tenkte meg noen enklere unit-tester som tester ulike typer av TekstFelt input i stedet

Reply:
Har egentlig ikke skrevet tester, (eller kode ;) ) ennå - vil mer diskutere konsept. Er dette noe vi ønsker å gjøre, eller skal vi prøve å løse det i UI? (som egentlig er bedre)

Reply:
Nytt forslag - hvordan parse T+Z datoer? 
Bruker substring. OK? 

Dersom noen senere skulle finne på å sende inn noe med f.eks. zone offset info kan det kanskje bli litt rart (da burde vi kanskje kastet exception?), men  jeg håper vi aldri skal trenge slikt.

Reply:
Startet med en versjon som parset litt diverse, men liker egentlig substring versjonen best. Veldig enkel og grei.

Reply:
Sånn - da har vi en versjon som oversetter 

2019-12-31:23:00:0Z -> 2020-01-01